tests/integration_tests/data_steward/validation/main_test.py

```diff
@@ -295,6 +296,31 @@ def test_html_report_five_person(self, mock_check_cron, mock_rdr_date,
         self.assertIn(main_consts.EHR_NO_PARTICIPANT_MATCH,
                       missing_record_types)
 
+        # test for required labs
```

Comment:
I'm not sure this code section should be merged as part of this PR.

Reply:
Yes, this was to add the failing test. Since we have #421, I'll remove these changes